Dockerfile

```diff
@@ -319,7 +319,7 @@ RUN curl -o nydus-static.tgz -fsSL --proto '=https' --tlsv1.2 "https://github.co
   mv nydus-static/nydus-image nydus-static/nydusd nydus-static/nydusify /usr/bin/ && \
   rm nydus-static.tgz
 CMD ["gotestsum", "--format=testname", "--rerun-fails=2", "--packages=github.com/containerd/nerdctl/v2/cmd/nerdctl/...", \
-  "--", "-timeout=30m", "-args", "-test.kill-daemon"]
+  "--", "-timeout=60m", "-args", "-test.kill-daemon"]
```

Comment:
Timeout is actually (also) controlled by github actions. Pushing this to 60 mins so that the Github Actions has actual effect.

Reply:
Next time please consider splitting this kind of change to another commit (or a PR)